regression-test/suites/datev2/tpcds_sf1_p1/sql/row_same_bit.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
```

Comment:
The SQL query and the issue you fixed seems unrelated, it's not suitable for a test case, we'd better design a simple clear case for the issue, so other developers can understand and modify the case easily.

Reply:
ok, i try some clear case, and add some comment for code